test/pkgs/Dynamo Samples/bin/SampleLibraryZeroTouch.Migrations.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0"?>
 <migrations>
   <priorNameHint>
-    <oldName>Examples.BasicExample.Create@Autodesk.DesignScript.Geometry.Point</oldName>
-    <newName>Examples.NEWBasicExample.Create@Autodesk.DesignScript.Geometry.Point</newName>
+    <oldName>Examples.BasicExample.Create</oldName>
+    <newName>Examples.NEWBasicExample.Create</newName>
```

Comment:
Should we add another test with the parameter names included just to make sure that works as well? 
I think we should also add a test to test migration to a new method with the same name but with new parameters?

Reply:
Theres an existing case below this one that tests your first requested case.

I'm not sure that your second requested case will actually work - if you change the parameters - this migration functionality has no way of wiring up the new parameters so they make sense... I can definitely try it out though and add a test.

Reply:
What about cases like this: https://github.com/DynamoDS/Dynamo/blob/d1195bea43e6d6f0570670b5ed85d1bf723e9427/extern/ProtoGeometry/ProtoGeometry.Migrations.xml#L77?

Reply:
And what about the case with the same namespace but different method name only?

Reply:
1. I think this magically works by reflection right? The order of the parameters is unchanged and only curve as become curve[] ?  - you are interested in a test that checks for this but with a migration that does not include parameter types in the migration right?

2. I can definitely add that.